modules/bitgo/test/v2/unit/coins/near.ts

```diff
@@ -554,4 +553,165 @@ describe('NEAR:', function () {
       }
     });
   });
+
+  describe('Parse Transactions:', () => {
```

Comment:
I'll elaborate my concern from test file even if it its actually related to code.

Can you add a test that has a huge amount as a number? because you allow from the TransactionOutput/Input numbers
`amount: number | string;` however Near has 24 decimal places, I think the posibility to broke this code is open.

Shouldnt be only string allowed? And why on L:306 we are parsing Input amount to string 
https://github.dev/BitGo/BitGoJS/blob/45c263271e8ff683dba4a8b3e8258fcb9b976414/modules/bitgo/src/v2/coins/near.ts#L306

However output amount could be both types, number or string?
https://github.dev/BitGo/BitGoJS/blob/45c263271e8ff683dba4a8b3e8258fcb9b976414/modules/bitgo/src/v2/coins/near.ts#L313

Another question, where basecoin.parseTransaction() is being used, it is a problem if we only return a string if this is the approach we are taking?

Reply:
We need only I think to consider string and not number.
Number of zeros is risky.

Reply:
@ipelaez-simtlix  as we discussed, I changed the tests to verify that they work with a large amount and also converted them to string.